jupyterlab_hub/typings/xterm/xterm.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for term.js 0.0.7
```

Comment:
However, if a typings file is not referenced in the `typings.d.ts` file, by our convention it's not used, so no need to copy it.